CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
Could you add an "Enhancements:" section and mention `IProxyTargetAccessor.DynProxySetTarget`.